ibm_catalog.json

```diff
@@ -19,7 +19,7 @@
         "ai"
       ],
       "short_description": "Deploy Cloud foundation for AI ops and governance with watsonx to automate the deployment and setup of the IBM watsonx platform",
-      "long_description": "The Cloud foundation for AI ops and governance with watsonx deployable architecture easily deploys and embeds AI across your business, manages all data sources, and accelerates responsible AI workflows on one platform.",
+      "long_description": "The Cloud foundation for AI ops and governance with watsonx deployable architecture easily deploys and embeds AI across your business, manages all data sources, and accelerates responsible AI workflows on one platform. \n\nLeverage [Terraform IBM Modules](https://github.com/terraform-ibm-modules) to shape and scale your solutions. You can integrate Terraform IBM Modules (TIM) to extend functionality and design a solution tailored to your environment and operations needs. These modules offer reusable, customizable elements that follow IBM Cloud's recommended practices. You can access the [source code and documentation](https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-saas-da) and use it to extend your current architecture or create new solutions.",
```

Comment:
There is no module in that repo - its just a DA. Perhaps we should link to the individual watsonx modules?